azure-pipelines.yml

```diff
@@ -194,7 +194,7 @@ extends:
     ACTest: ${{ parameters.RunACTests }}
     releaseParameters:
       environment: Preview
-      netVersion: 3.1.x
+      dotNetCoreVersion: '6.0'
```

Comment:
this should say `netVersion: '6.x'`

It also needs to be in the `testParameters` section, not the `releaseParameters` section. Use the notification API file as a reference: https://github.com/hmcts/vh-notification-api/blob/master/azure-pipelines.yml#L317